### PR TITLE
Update image and add description for "Allow remote shopping assistance" and "Show Password"

### PR DESCRIPTION
**Pull request by pawan**
## Purpose of this pull request

Fix for #1449 

## Magento release version

2.4

## Product editions

Magento Open Source 
Adobe Commerce

## Affected documentation pages

https://docs.magento.com/user-guide/customers/account-create.html

whatsnew
Updated the [Creating an Individual Customer Account](https://docs.magento.com/user-guide/customers/account-create.html) topic to include the _Allow remote shopping assistance_ and _Show Password_ options available to customers on the storefront.

### CODE REVIEW

File: src/customers/account-create.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ A store customer creates a new account on the storefront.
 
     This option appears even if the store doesn’t publish a newsletter.
 
+1. If you want to allows merchants to [See what you see]({% link customers/login-as-customer.md %}) and take actions on your behalf in order to provide better assistance, select the **Allow remote shopping assistance** checkbox.
```
Comment (leslietilling) [inline src/customers/account-create.md:32]:
```suggestion
1. If you want to enable store support staff to [see what you see]({% link customers/login-as-customer.md %}) and take actions on your behalf in order to provide better assistance, select the **Allow remote shopping assistance** checkbox.
```

File: src/customers/account-create.md [hunk 2]
```diff
@@ -49,6 +51,8 @@ A store customer creates a new account on the storefront.
 
     Then, enter it again to **Confirm Password**.
 
+1. Click <span class="btn">Show Password</span> to see what password you are entering.
```
Comment (leslietilling) [inline src/customers/account-create.md:54]:
```suggestion
1. If needed, click <span class="btn">Show Password</span> to see view the password you entered.
```

**Review by leslietilling — APPROVED** [review]

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just a couple of edits

**Comment by pawan** [comment]
> Just a couple of edits

Updated @leslietilling

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**